routes/admin.js

```diff
@@ -0,0 +1,56 @@
+const route = require('express').Router();
+const bcrypt = require('bcrypt');
+
+const { Admin } = require('../database/models/admin'); 
+
+route.post('/register', async (req, res) => {
+    try {
+      const {
+        register_token,
+      } = req.query;
+      if (register_token !== process.env.REGISTER_TOKEN) {
+        return res.status(403).json({ message: 'Incorrect token' });
+      }
+
+      const {
+        login,
+        password,
+      } = req.body;
+
+      const adminExist = await Admin.findOne({ login });
+      if (adminExist) {
+        return res.status(400).json({ message: 'User with this login already exists' });
+      }
+
+      await bcrypt.genSalt(10, (err, salt) => {
+        if (err) {
+          return res.status(500).json('crypt crashed gen salt');
```

Comment:
никогда не возвращай строчку

```suggestion
return res.status(500).json({
  success: false,
  message: 'crypt crashed while generating salt',
});
```